README.md

```diff
@@ -4,16 +4,43 @@ Git bot for sending notification about bugzilla new bugs
 
 ## Config
 
-By file `config.yml` with data
+The `config.yml` file must have the following structure:
 
 ```yaml
-telegram_bot_token: token
-channel_id: id-of-channel
-bugzilla_url: bugzilla_url
-bugzilla_key: bugzilla_key
-check_period: 60 # Timeout between checks for new bug
+chat1:
+  telegram_bot_token: token
```

Comment:
Cant' we use same token for both chat?

Correct me if I wrong - didn't the token is like 'app specific password for bot to send notifications'
And since we use same bot account in both chats it should be same token?

Channel_id for sure I understand will be different

Reply:
Decided to have different tokens are more flexible